src/utils/junit/JUnitCommandHandler.ts

```diff
@@ -91,6 +92,7 @@ export class JUnitCommandHandler {
 			}
 			loader.stop()
 		} catch (e) {
+			loader.stop()
```

Comment:
Fixes error message being on the same line:

![image](https://github.com/Hypersequent/qas-cli/assets/18312847/14ba7d56-382f-4566-952c-a1776f3a6f1a)